workers/go/workflows/utils.go

```diff
@@ -0,0 +1,29 @@
+package workflows
+
+import (
+	"fmt"
+	"go.temporal.io/sdk/workflow"
+)
+
+// RunConcurrently runs the given functions concurrently, waiting for all to complete.
+// Returns early if any of the functions return an error.
+func RunConcurrently(ctx workflow.Context, funcs ...func(workflow.Context) error) error {
+	completed := 0
+	var innerErr error
+	for _, f := range funcs {
+		f := f
+		workflow.Go(ctx, func(ctx workflow.Context) {
+			innerErr = f(ctx)
```

Comment:
This could swallow an error.

```suggestion
			if err := f(ctx); err != nil {
				innerErr = err
			}
```

Reply:
Thx